module-services/service-fileindexer/ServiceFileIndexer.cpp

```diff
@@ -14,13 +14,14 @@ namespace
     {
         return purefs::createPath(purefs::dir::getUserDiskPath(), "music").string();
     }
+    inline constexpr auto fileIndexerServiceStackSize = 1024 * 5;
```

Comment:
it's actually a pretty big bump, what happened?

Reply:
Currently, only 7% of the space remains. We need more stock - I set it to at least 20%.
![stackstats](https://user-images.githubusercontent.com/70628239/166897679-169c5e8b-a510-4a57-81fd-71cd03cfdabe.png)